### ISSUE TITLE
Where's the Mac version ? 

### ISSUE THREAD

**Issue by Prattjames** [ZeroX-DG/SnippetStore#5]
Hi Guys,

This is a fantastic Idea, it can even be more powerful combined with VS Code plugin to directly put a snippet in the App while coding.

The issue is that I don't see the Mac version... or it's not available for Mac users ?

**Comment by ZeroX-DG** [comment]
I'm sorry @Prattjames but as I said in the download section in the website, SnippetStore developer team has only 1 member and I can't test SnippetStore on MacOS because I use linux and poor and can't afford a Mac :smile: . But I'll try to find someone who use Mac to work with me. In the meantime, you can build SnippetStore from source (not guarantee it will work) using
```
npm run dist:mac
```

**Comment by Prattjames** [comment]
@ZeroX-DG Everything worked just fine.
For people that needs the DMG file [https://ufile.io/gqb53](https://ufile.io/gqb53).

**Comment by Prattjames** [comment]
@prajwalrao you're right, I can't also, there's a bug in the mac version I think, we can't copy/paste...

**Comment by ZeroX-DG** [comment]
I think I know this bug but I can't test my solution for this bug. But I'll add it anyway. If you want, you can donate me through [buy me a coffee](https://www.buymeacoffee.com/hQteV8A) so that I can afford a mac to test & build SnippetStore on mac and many other open source projects in the future.

**Comment by LauGau** [comment]
Up ! I will definitely buy you a coffee (even a pack !) @ZeroX-DG if there is a mac version :D

**Comment by ZeroX-DG** [comment]
@LauGau thanks for the offer but I don't think I'll be able to create any mac version anytime soon. Although, I might be able to do that if my next job provide me with a mac :smile: In the mean time, you can clone the repo and build it yourself:)
